plugins/search-backend-node/api-report.md

```diff
@@ -67,4 +118,21 @@ export class Scheduler {
 }
 
 export { SearchEngine };
+
+// Warning: (ae-missing-release-tag) "TestPipeline" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export class TestPipeline {
```

Comment:
Hmm...  Does it make sense to pop in `@backstage/test-utils`?  To avoid the dependency from other plugin backends on `@backstage/plugin-search-backend-node`?

Reply:
Would prolly say it's best to keep in `-node` tbh, since it's very specific to the search infra. Other plugin backends can have it as a devDep if needed, although tbh I would assume that if they're interested in testing things with this they'd already need it as a regular dependency?